pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -245,6 +247,7 @@ workflow Optimus {
     File gene_metrics = GeneMetrics.gene_metrics
     File? cell_calls = RunEmptyDrops.empty_drops_result
     File? picard_metrics = DropseqMetrics.metric_output
+    File? aligner_metrics = MergeStarOutputs.cell_reads_out
```

Comment:
Hey Liz - would you mind adding the new output to the [Optimus Overview doc Outputs table](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Optimus_Pipeline/README.md#9-outputs)?

Reply:
Thanks @kayleemathews ! Added it.

Reply:
Thank you!